dask/array/tests/test_fft.py

```diff
@@ -111,6 +111,7 @@ def test_wrap_bad_kind():
         fft_wrap(np.ones)
 
 
+@pytest.mark.slow
 @pytest.mark.parametrize("funcname", all_nd_funcnames)
 @pytest.mark.parametrize("dtype", ["float32", "float64"])
 def test_nd_ffts_axes(funcname, dtype):
```

Comment:
Maybe a better fix for this one is to drop a dimension. This is using 4 dims to test with, but 3 is probably just as instructive. Besides if it is going to end up being skipped anyways, better to come up with a leaner version that we can still run.